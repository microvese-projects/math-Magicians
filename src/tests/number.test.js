import React from 'react';
import renderer from 'react-test-renderer';
import Number from '../components/number';

test('Should render calculator correctly', () => {
  const tree = renderer
    .create(<Number />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
