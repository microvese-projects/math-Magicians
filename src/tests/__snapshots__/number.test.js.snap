// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render calculator correctly 1`] = `
<button
  onClick={[Function]}
  type="button"
/>
`;
